src/table/table-head.component.ts

```diff
@@ -0,0 +1,242 @@
+import {
+	ApplicationRef,
+	Component,
+	HostBinding,
+	Input,
+	Output,
+	EventEmitter
+} from "@angular/core";
+
+import { TableModel } from "./table.module";
+import { I18n } from "./../i18n/i18n.module";
+
+@Component({
+	// tslint:disable-next-line:component-selector
+	selector: "[ibm-table-head]",
```

Comment:
Should be 
```suggestion
	selector: "[ibmTableHead]",
```

since we're following the directive pattern